demosplan/DemosPlanCoreBundle/Logic/Statement/StatementService.php

```diff
@@ -2745,27 +2746,21 @@ public function bulkEditStatementsAddData(StatementBulkEditVO $statementEdit): v
      */
     public function getPhaseNameFromArray(array $statement): string
     {
+        $statementObject = $this->getStatement($statement['id']);
+
         return $this->getPhaseName(
             $statement['phase'],
-            $statement['publicStatement']
+            $statementObject->isSubmittedByCitizen()
         );
     }
 
-    public function getPhaseName(string $phaseKey, string $publicStatement): string
+    public function getPhaseName(string $phaseKey, bool $isSubmittedByCitizen): string
```

Comment:
Can we rename this function to getProcedurePhaseName?

Reply:
done